.travis.yml

```diff
@@ -16,7 +16,7 @@ matrix:
       env: LANG=en_US.UTF-8
     - name: "Linting"
       script:
-        - pip install black==19.3b0
```

Comment:
:disappointed: This does make me reconsider the 3.5 choice to be honest. I didn't think we were losing much by using it, but I note that Django 3 won't support it either so maybe it's worth reconsidering.